html-tables/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-tables</title>
+</head>
+
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+    <thead>
+    <tr></tr>
+    <th scope="col">Student Name</th>
+    <th scope="col">Course</th>
+    <th scope="col">Grade</th>
+    </thead>
+    <tr></tr>
+    <tbody>
+      <td>Grace Hopper</td>
+      <td>Computer Science</td>
+      <td>100</td>
+    <tr></tr>
```

Comment:
as noted above, make sure to have a `tr` for each row with the `td`s as children